[🐛 BUG]: rr jobs command throws runtime error

### No duplicates 🥲.

- [X] I have searched for a similar issue in our bug tracker and didn't find any solutions.

### What happened?

I want to see jobs statistic for RR server with jobs plugin.

### Version (rr --version)

rr version 2.12.3 (build time: 2023-02-16T13:08:41+0000, go1.20), OS: linux, arch: amd64

### How to reproduce the issue?

.rr.yaml simple config

```
version: "2.7"

rpc:
    listen: tcp://127.0.0.1:6001

server:
    command: php public/worker.php

amqp:
    addr: amqp://guest:guest@rabbitmq:5672

logs:
    level: debug
    mode: development
    output: stderr

jobs:
    num_pollers: 3

    pool:
        num_workers: 10

    consume: [ "test-1"]

    pipelines:
        test:
            driver: amqp
            config:
                queue: test-1
                exchange: test
```

run `./rr serve -c .rr.yaml`

run `./rr jobs` **and got runtime error**

run `./rr workers` **and see full information without error**

### Relevant log output

```shell
/var/www # ./rr jobs
panic: runtime error: index out of range [0] with length 0

goroutine 1 [running]:
github.com/roadrunner-server/roadrunner/v2/internal/cli/jobs.NewCommand.func1(0xc0000d6f00?, {0x2bd1740, 0x0, 0x0?})
        github.com/roadrunner-server/roadrunner/v2/internal/cli/jobs/command.go:39 +0x4db
github.com/spf13/cobra.(*Command).execute(0xc0000d6f00, {0x2bd1740, 0x0, 0x0})
        github.com/spf13/cobra@v1.6.1/command.go:916 +0x862
github.com/spf13/cobra.(*Command).ExecuteC(0xc0000d6000)
        github.com/spf13/cobra@v1.6.1/command.go:1044 +0x3bd
github.com/spf13/cobra.(*Command).Execute(0x7ffc018dacd2?)
        github.com/spf13/cobra@v1.6.1/command.go:1044 +0x3bd
github.com/spf13/cobra.(*Command).Execute(0x7fff17263cd2?)
        github.com/spf13/cobra@v1.6.1/command.go:968 +0x19
main.run()
        github.com/roadrunner-server/roadrunner/v2/cmd/rr/main.go:26 +0x45
main.main()
        github.com/roadrunner-server/roadrunner/v2/cmd/rr/main.go:16 +0x19

/var/www # ./rr workers
Workers of [jobs]:
+---------+-----------+---------+---------+---------+--------------------+
|   PID   |  STATUS   |  EXECS  | MEMORY  |  CPU%   |      CREATED       |
+---------+-----------+---------+---------+---------+--------------------+
|     277 | ready     |       0 | 54 MB   |    0.22 | 1 minute ago       |
|     279 | ready     |       0 | 54 MB   |    0.20 | 1 minute ago       |
|     270 | ready     |       0 | 54 MB   |    0.21 | 1 minute ago       |
|     268 | ready     |       0 | 54 MB   |    0.22 | 1 minute ago       |
|     280 | ready     |       0 | 54 MB   |    0.22 | 1 minute ago       |
|     273 | ready     |       0 | 54 MB   |    0.22 | 1 minute ago       |
|     272 | ready     |       0 | 54 MB   |    0.22 | 1 minute ago       |
|     276 | ready     |       0 | 54 MB   |    0.18 | 1 minute ago       |
|     274 | ready     |       0 | 54 MB   |    0.20 | 1 minute ago       |
|     278 | ready     |       0 | 54 MB   |    0.21 | 1 minute ago       |
+---------+-----------+---------+---------+---------+--------------------+
Jobs of [jobs]:
+----------------+----------+--------+--------+--------+---------+----------+
|     STATUS     | PIPELINE | DRIVER | QUEUE  | ACTIVE | DELAYED | RESERVED |
+----------------+----------+--------+--------+--------+---------+----------+
| PAUSED/STOPPED | test     | amqp   | test-1 | 0      | 0       | 0        |
+----------------+----------+--------+--------+--------+---------+----------+
```

--- a/.golangci.yml
+++ b/.golangci.yml
@@ -54,7 +54,6 @@ linters: # All available linters list: <https://golangci-lint.run/usage/linters/
     - errorlint # find code that will cause problems with the error wrapping scheme introduced in Go 1.13
     - exhaustive # check exhaustiveness of enum switch statements
     - exportloopref # checks for pointers to enclosing loop variables
-    - funlen # Tool for detection of long functions
     - gochecknoglobals # Checks that no globals are present in Go code
     - gocognit # Computes and checks the cognitive complexity of functions
     - goconst # Finds repeated strings that could be replaced by a constant

--- a/internal/cli/jobs/command.go
+++ b/internal/cli/jobs/command.go
@@ -35,6 +35,10 @@ func NewCommand(cfgFile *string, override *[]string, silent *bool) *cobra.Comman
 				return errors.E(op, errors.Str("no configuration file provided"))
 			}
 
+			if len(args) == 0 {
+				return errors.Str("incorrect command usage, should be: rr jobs pause/resume/destroy/list")
+			}
+
 			// for the commands other than list, args[1] should contain list of pipelines to pause/resume/destroy
 			if !listPipes && len(args[0]) == 0 {
 				return errors.Str("pause/resume/destroy commands should have list of the pipelines as second arg")

